scripts/buildtools/commands/prepack.js

```diff
@@ -20,6 +20,11 @@ exports.handler = async () => {
     'src'
   ], `${BUILD_DIR}`);
 
+  // remove node_modules/ from package in case it gets copied in previous step
+  shell.rm('-Rf', [
+    `${BUILD_DIR}/src/v3/node_modules/`,
+  ]);
```

Comment:
I couldn't find a way to prevent this directory from being copied in the previous `shell.cp` step but this is the next best thing. It's just a bit of a waste of time to copy the `node_modules/` directory just to delete it (which also takes a bit of time).